ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -27,6 +27,7 @@ open class CommandBarItem: NSObject {
         self.isEnabled = isEnabled
         self.isSelected = isSelected
         self.itemTappedHandler = itemTappedHandler
+        self.isHidden = false
```

Comment:
Minor: Can initialize in place in the var declaration `    @objc public var isHidden = false {`